### PR TITLE
Buildpack troubleshooting tip

### PR DESCRIPTION
**Pull request by katarzyna-koltun-mx**

### CODE REVIEW

File: content/en/docs/developerportal/deploy/cloud-foundry-deploy.md [hunk 1]
```diff
@@ -185,6 +185,18 @@ If you encounter any problems, you should consult the application logs. These ar
...
+
+When using the buildpack to push a Mendix project folder to a Cloud Foundry cluster, the push fails.
+
+#### 8.1.1 Cause
+
+Pushing the project folder to a Cloud Foundry cluster is not currently supported.
+
+#### 8.1.2 Solution
+
+This issue will be solved in an upcoming version of the buildpack.
```
Comment (feddovanede) [inline content/en/docs/developerportal/deploy/cloud-foundry-deploy.md:204]: We could add workarounds:

1. If the app model is stored on the Mendix Teamserver: use the Deploy API to build a deployment package and push that with the CF buildpack
2. Build a deployment package from Studio Pro and push that with the CF buildpack
3. Run MxBuild from your CI/CD pipeline to build a deployment package and push that with the CF buildpack

Comment (feddovanede) [reply content/en/docs/developerportal/deploy/cloud-foundry-deploy.md:204]: 1. https://docs.mendix.com/apidocs-mxsdk/apidocs/build-api/#start-building-deployment-package
2. https://docs.mendix.com/refguide/app-menu/#6-create-deployment-package
3. https://docs.mendix.com/refguide/mxbuild/

File: content/en/docs/developerportal/deploy/cloud-foundry-deploy.md [hunk 2]
```diff
@@ -185,6 +185,18 @@ If you encounter any problems, you should consult the application logs. These ar
 
 {{< figure src="/attachments/developerportal/deploy/cloud-foundry-deploy/cf-log.png" >}}
 
+### 8.1 Cannot Push Project Folder
+
+When using the buildpack to push a Mendix project folder to a Cloud Foundry cluster, the push fails.
```
Comment (feddovanede) [inline content/en/docs/developerportal/deploy/cloud-foundry-deploy.md]: According to [this ticket](https://mendixsupport.zendesk.com/agent/tickets/190746), the error message is
`Cannot open assembly 'MendixBuilder/modeler/mxbuild.exe': no such file or directory.`

**Review by feddovanede — COMMENTED** [review]
Added error message. We could add workarounds (in some cases solutions, as we will not fix this issue in all cases).

File: content/en/docs/developerportal/deploy/cloud-foundry-deploy.md (hunk 1 above)
Comment (katarzyna-koltun-mx) [reply content/en/docs/developerportal/deploy/cloud-foundry-deploy.md:204]: Done.

File: content/en/docs/developerportal/deploy/cloud-foundry-deploy.md (hunk 2 above)
Comment (katarzyna-koltun-mx) [reply content/en/docs/developerportal/deploy/cloud-foundry-deploy.md]: Done.

**Comment by katarzyna-koltun-mx** [comment]
@feddovanede I've made the requested corrections, shall I publish the file now?

**Comment by feddovanede** [comment]
@katarzyna-koltun-mx One small remark: I think it is the _Build API_ that is used to build a deployment package, not the Deploy API. Besides that, looks ok to me!

**Comment by katarzyna-koltun-mx** [comment]
@feddovanede Makes sense, thank you. :D Done and merging.

**Pull request merged**